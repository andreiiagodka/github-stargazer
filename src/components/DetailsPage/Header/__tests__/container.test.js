import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'

import HeaderContainer from '..'

describe('<HeaderContainer />', () => {
  it('renders correctly', () => {
    const mockStore = configureStore([])
    const store = mockStore({})

    const repository = { full_name: 'full name' }
    const history = {}

    const props = { repository: repository, history: history,  }
    const wrapper = shallow(<HeaderContainer.WrappedComponent store={store} {...props} />)
    const container = wrapper.dive()

    expect(container).toMatchSnapshot()
  })
})
