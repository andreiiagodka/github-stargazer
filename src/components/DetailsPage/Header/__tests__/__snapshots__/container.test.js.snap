// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderContainer /> renders correctly 1`] = `
<div
  className="row"
>
  <Col
    md={2}
  />
  <Col
    className="text-center"
    md={8}
  >
    <h5>
      full name
    </h5>
  </Col>
  <Col
    className="text-right"
    md={2}
  >
    <Button
      active={false}
      disabled={false}
      onClick={[Function]}
      variant="light"
    >
      <FontAwesomeIcon
        border={false}
        className=""
        fixedWidth={false}
        flip={null}
        icon={
          Object {
            "icon": Array [
              448,
              512,
              Array [],
              "f1f8",
              "M432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16zM53.2 467a48 48 0 0 0 47.9 45h245.8a48 48 0 0 0 47.9-45L416 128H32z",
            ],
            "iconName": "trash",
            "prefix": "fas",
          }
        }
        inverse={false}
        listItem={false}
        mask={null}
        pull={null}
        pulse={false}
        rotation={null}
        size={null}
        spin={false}
        swapOpacity={false}
        symbol={false}
        title=""
        transform={null}
      />
    </Button>
  </Col>
</div>
`;
